WebDriverAgentLib/Commands/FBCustomCommands.m

```diff
@@ -227,4 +229,20 @@ + (BOOL)isKeyboardPresent {
   return FBResponseWithOK();
 }
 
++ (id <FBResponsePayload>)handleLaunchUnattachedApp:(FBRouteRequest *)request
+{
+  NSString *bundle = (NSString *)request.arguments[@"bundleId"];
+  if ([[LSApplicationWorkspace defaultWorkspace] openApplicationWithBundleID:bundle])
+    return FBResponseWithOK();
+  return FBResponseWithError([self appLaunchError]);
+}
+
++ (NSError *)appLaunchError
```

Comment:
do we really need a separate method to build and error? Also, WDA has an error builder class

Reply:
WDA has an error builder class => `FBErrorBuilder`

Reply:
Fixed